blog/boa-release-21/index.md

```diff
@@ -0,0 +1,221 @@
+in conformance will be minor or bound to the feature size going forward.
+
+You can check the full list of changes [here][changelog], and the full
+information on conformance [here][conformance].
+
+<!--truncate-->
+
+## Feature Highlights
+
+### Temporal
+
+There has been a lot of progress made on Temporal, the new Stage 3
+date/time proposal. With this release, Boa's conformance on Temporal
+grew from 40.67% to 97.06%. As of this moment, Boa is the second most
+conformant Temporal implementation behind SpiderMonkey with others soon
+to follow.
+
+The implementation is backed by the `temporal_rs` date/time Rust
+library, which we went over briefly in our June
+[blog post](./2025-06-15-temporal-impl-1.md) with hopefully another post
+in the not too distant future. So far, `temporal_rs` has been used in
+both V8 and Keisel to implement Temporal as well.
+
+Temporal can be used from `boa_cli` or enabled in `boa_engine` with the
+`experimental` or `temporal` feature.
+
+You can also try out Temporal directly in Rust using `temporal_rs`:
+
+```
+cargo add temporal_rs
+```
+
+#### Special thanks
+
+Special thanks to all the student's from University of Bergen who helped contribute to
+`temporal_rs` and Boa, as well as Shane Carr ([@sffc](https://github.com/@sffc)) and
+Manish Goregaokar ([@manishearth](https://github.com/@manishearth)) for their contributions
+and help with `temporal_rs`.
+
+### Span nodes and error backtraces
+
+We also closed an outstanding issue of supporting span nodes in our parser and AST. This
+allowed us to implement backtraces to provide better debugging information when using
+Boa.
+
+![backtrace-example](./img/backtrace.gif)
+
+### New Set methods
+
+This release adds support for the new set methods added in ES2025.
+
+The new methods are:
+
+- `Set.prototype.intersection(other)`
+- `Set.prototype.union(other)`
+- `Set.prototype.difference(other)`
+- `Set.prototype.symmetricDifference(other)`
+- `Set.prototype.isSubsetOf(other)`
+- `Set.prototype.isSupersetOf(other)`
+- `Set.prototype.isDisjointFrom(other)`
+
+Thanks to [@Hemenguelbindi](https://github.com/@Hemenguelbindi) for
+their work on this feature.
+
+### Float16 support for TypedArrays, Dataview and Math built-ins
+
+This release adds support for `f16` types for the TypedArray, Dataview, and Math
+built-ins.
+
+```javascript
+const x = new Float16Array([37, 42.123456]);
+console.log(x[1]); // 42.125
+```
+
+### Error.isError
+
+This release adds support for `Error.isError`, which is accepted for ES2026.
+
+```javascript
+let check = Error.isError(new Error()); // returns true
+```
+
+### Math.sumPrecise
+
+This release adds support for `Math.sumPrecise`, which is accepted for ES2026.
+
+We've opted for using the new [`xsum`](https://crates.io/crates/xsum) summation algorithm
+for the underlying implementation.
+
+```javascript
+let sum = Math.sumPrecise([1e20, 0.1, -1e20]);
+console.log(sum); // 0.1
+```
```

Comment:
We should potentially group any small spec additions into a separate section. It makes the post look a bit cleaner and avoids having too many header titles